src/client/src/Web/index.tsx

```diff
@@ -33,142 +40,123 @@ const TornOut = ({ children }: WithChildren) => (
-                </TornOut>
-              )
-            }}
+            element={
+              <TornOut>
+                <CreditRfqForm />
+              </TornOut>
+            }
           />
         )}
         {canDisplayCredit && (
           <Route
             path={ROUTES_CONFIG.creditBlotter}
-            render={() => {
-              return (
-                <TornOut>
-                  <CreditTrades />
-                </TornOut>
-              )
-            }}
+            element={
+              <TornOut>
+                <CreditTrades />
+              </TornOut>
+            }
           />
         )}
-        <Route
-          path={ROUTES_CONFIG.tile}
-          render={({
-            location: { search },
-            match: {
-              params: { symbol },
-            },
-          }) => {
-            const query = new URLSearchParams(search)
-            const view = query.has("tileView")
-              ? (query.get("tileView") as TileView)
-              : TileView.Analytics
-
-            return (
-              <>
-                <DisconnectionOverlay />
-                <TearOutRouteWrapper>
-                  {symbol && <TornOutTile symbol={symbol} view={view} />}
-                </TearOutRouteWrapper>
-              </>
-            )
-          }}
-        />
+        <Route path={ROUTES_CONFIG.tile} element={<TileWrapper />} />
 
         <Route
           path={ROUTES_CONFIG.styleguide}
-          render={() => (
+          element={
             <Suspense fallback={<Loader />}>
               <StyleguidePage />
             </Suspense>
-          )}
+          }
         />
 
         {canUseAdmin && (
-          <Route path={ROUTES_CONFIG.admin} render={() => <Admin />} />
+          <Route path={ROUTES_CONFIG.admin} element={<Admin />} />
         )}
 
         <Route
           path={ROUTES_CONFIG.tiles}
-          render={() => {
-            return (
-              <TornOut>
-                <LiveRates />
-              </TornOut>
-            )
-          }}
+          element={
+            <TornOut>
+              <LiveRates />
+            </TornOut>
+          }
         />
         <Route
           path={ROUTES_CONFIG.blotter}
-          render={() => {
-            return (
-              <TornOut>
-                <FxTrades />
-              </TornOut>
-            )
-          }}
+          element={
+            <TornOut>
+              <FxTrades />
+            </TornOut>
+          }
         />
         <Route
           path={ROUTES_CONFIG.analytics}
-          render={() => {
-            return (
-              <TornOut>
-                <Analytics hideIfMatches={""} />
-              </TornOut>
-            )
-          }}
+          element={
+            <TornOut>
+              <Analytics hideIfMatches={""} />
+            </TornOut>
+          }
         />
-        <Redirect to="/" />
-      </Switch>
+        <Route path="*" element={<Navigate to="/" />} />
```

Comment:
nice .. have to thrash this, I had all sorts of funky issues with previous react-router, as you know